src/Charts/DoughnutChart/DoughnutChart.js

```diff
@@ -1,24 +1,24 @@
 import React, { Component } from 'react'
 import { Doughnut } from 'react-chartjs'
 import { colors } from '../../tools.js'
+import DoughnutChartList from './DoughnutChartList/DoughnutChartList'
 import Percent from '../../Statistics/Percent/Percent'
-import MediaQuery from 'react-responsive'
 
 class DoughnutChart extends Component {
-  getData() {
-    const data = this.props.data
-
-    return Object.keys(data).map( (val, idx) => {
-      const color = this.setColor(idx)
-
-      return {
-        'label': val,
-        'value': data[val],
-        'color':  color.value,
-        'colorName': color.name
-      } })
+  formatData(data) {
```

Comment:
`formatData` and `setColor` have nonbenefit to be attached to the component. You can use pure function.